src/Commands/AdminCommand.php

```diff
@@ -2,12 +2,13 @@
 
 namespace TCG\Voyager\Commands;
 
+use App\User;
+use Illuminate\Support\Str;
```

Comment:
Please avoid committing changes like these (rearranging "use" statements).  They provide no benefit, and simply make StyleCI unhappy.  I realize it likely wasn't intentional, so don't worry about it.  StyleCI will auto-fix it after merge anyway.